app-frontend/src/app/services/scenes/planetRepository.service.js

```diff
@@ -56,6 +56,7 @@ export default (app) => {
                         reject();
                     });
                 }
+                this.thumbnailSize = 512;
```

Comment:
this is basically a constant, so set it in the constructor. InitRepository is used to do operations that have to happen inside a promise